source/sass/buyers-guide/views/product.scss

```diff
@@ -134,6 +134,19 @@
     }
   }
 
+  .company-external-link {
+    display: inline-flex;
+    align-items: center;
+
+    &::after {
+      content: " ";
+      width: 17px;
+      height: 16px;
+      background: url(../_images/buyers-guide/external-link.svg) no-repeat;
+      margin-left: 8px;
```

Comment:
this feels very close to just `width: 1em`, `height: 1em` and `margin-left: 0.5em`, which probably makes more sense if we're setting the icon based on the text it goes with. @fessehaye, thoughts?

Reply:
Hi @Pomax! Awesome catch, I agree that the original sizing was very close to 1em and 0.5em. 
I have updated the css to use the EM values now, and can also confirm that they solve the icon cutoff issue from above as well.